tasks/fpm.yml

```diff
@@ -1,19 +1,28 @@
 ---
 
-- name: "Install PHP-FPM package: {{ 'present' if php_fpm_enable else 'absent' }}"
+- name: Install PHP-FPM package
+  become: yes
   apt:
     name: "php{{ php_version }}-fpm"
-    state: "{{ 'present' if php_fpm_enable else 'absent' }}"
+    state: present
```

Comment:
These are all missing the `php-fpm` tag so that we can target them. I think a good practice to get into is to tag a task by the service and also by the main role. Since this is relating to php-fpm, but is part of the php role we can tag it both by php and php-fpm, giving us two options to target.

This idea should apply to php-cli as well.

Reply:
On second thought; does this behavior already exist since you're using the include task with the proper tags?

Reply:
Yes, everything that gets included gets run if the tag on the `include` statement is used.